scripts/build_backend.py

```diff
@@ -0,0 +1,90 @@
+import os
+import sys
+import subprocess
+import shutil
+import glob
+
+IS_WIN = False
+IS_LIN = False
+
+if "linux" in sys.platform:
+    IS_LIN = True
+elif sys.platform in ["win32", "cygwin"]:
+    IS_WIN = True
+else:
+    assert False, sys.platform + " not supported"
+
+ONEAPI_ROOT = os.environ.get("ONEAPI_ROOT")
+
+if IS_LIN:
+    DPCPP_ROOT = os.path.join(ONEAPI_ROOT, "compiler/latest/linux")
+if IS_WIN:
+    DPCPP_ROOT = os.path.join(ONEAPI_ROOT, "compiler\latest\windows")
+
+dpctl_dir = os.getcwd()
+build_cmake_dir = os.path.join(dpctl_dir, "build_cmake")
+if os.path.exists(build_cmake_dir):
+    shutil.rmtree(build_cmake_dir)
+os.mkdir(build_cmake_dir)
+os.chdir(build_cmake_dir)
+
+INSTALL_PREFIX = os.path.join(dpctl_dir, "install")
+if os.path.exists(INSTALL_PREFIX):
+    shutil.rmtree(INSTALL_PREFIX)
+
+backends_dir = os.path.join(dpctl_dir, "backends")
+
+if IS_LIN:
+    subprocess.check_call(
+        [
+            "cmake",
+            "-DCMAKE_BUILD_TYPE=Release",
+            "-DCMAKE_INSTALL_PREFIX=" + INSTALL_PREFIX,
+            "-DCMAKE_PREFIX_PATH=" + INSTALL_PREFIX,
+            "-DDPCPP_ROOT=" + DPCPP_ROOT,
+            "-DCMAKE_C_COMPILER:PATH=" + os.path.join(DPCPP_ROOT, "bin", "clang"),
+            "-DCMAKE_CXX_COMPILER:PATH=" + os.path.join(DPCPP_ROOT, "bin", "dpcpp"),
+            backends_dir,
```

Comment:
Make a list var please with all args and send the list to subprocess call. Also using `f"-DCMAKE_INSTALL_PREFIX={INSTALL_PREFIX}"` is shorter.